ts/components/core/accordion/RawAccordion.tsx

```diff
@@ -0,0 +1,125 @@
+import { View } from "native-base";
+import * as React from "react";
+import { useEffect, useRef, useState } from "react";
+import {
+  AccessibilityProps,
+  Animated,
+  Easing,
+  LayoutAnimation,
+  Platform,
+  StyleSheet,
+  TouchableOpacity,
+  UIManager
+} from "react-native";
+import I18n from "../../../i18n";
+import themeVariables from "../../../theme/variables";
+import customVariables from "../../../theme/variables";
+import IconFont from "../../ui/IconFont";
+import { IOStyles } from "../variables/IOStyles";
+
+// TODO: handle external initial open/closed state
+type Props = {
+  // The header component, an arrow indicating the open/closed state will be added on the right
+  header: React.ReactElement;
+  // The accordion component must accept one children
+  children: React.ReactElement;
+  // The component should be animated?
+  animated?: boolean;
+  headerStyle?: React.ComponentProps<typeof View>["style"];
+  accessibilityLabel?: AccessibilityProps["accessibilityLabel"];
+};
+
+const styles = StyleSheet.create({
+  headerIcon: {
+    alignSelf: "center"
+  },
+  row: {
+    ...IOStyles.row,
+    justifyContent: "space-between"
+  },
+  internalHeader: {
+    flex: 1,
+    paddingRight: themeVariables.contentPadding
+  }
+});
+
+/**
+ * Obtains the degree starting from the open state
+ * @param isOpen
+ */
+const getDegree = (isOpen: boolean) => (isOpen ? "-90deg" : "-270deg");
+
+/**
+ * The base accordion component, implements the opening and closing logic for viewing the children
+ * @param props
+ * @constructor
+ */
+export const RawAccordion: React.FunctionComponent<Props> = props => {
+  const [isOpen, setOpen] = useState<boolean>(false);
+  const animatedController = useRef(new Animated.Value(1)).current;
+  const shouldAnimate = props.animated ?? true;
+  const headerStyle = props.headerStyle ?? {};
+  const accessibilityLabel = props.accessibilityLabel
+    ? `${props.accessibilityLabel}, `
+    : "";
+
+  const arrowAngle = shouldAnimate
+    ? animatedController.interpolate({
+        inputRange: [0, 1],
+        outputRange: ["-90deg", "-270deg"]
+      })
+    : getDegree(isOpen);
+
+  useEffect(() => {
+    if (Platform.OS === "android") {
+      UIManager.setLayoutAnimationEnabledExperimental(shouldAnimate);
+    }
+  }, [shouldAnimate]);
+
+  const onPress = () => {
+    if (shouldAnimate) {
+      LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut);
+      Animated.timing(animatedController, {
+        duration: 300,
+        toValue: isOpen ? 1 : 0,
+        useNativeDriver: true,
+        easing: Easing.linear
+      }).start();
+    }
+    setOpen(!isOpen);
+  };
+
+  return (
+    <View style={IOStyles.flex}>
+      <TouchableOpacity
```

Comment:
```suggestion
      <TouchableDefaultOpacity
```
To avoid this effect

https://user-images.githubusercontent.com/822471/147571753-ad0ba508-6a3b-42ee-8b12-8022b263891f.mov

this is why https://www.pivotaltracker.com/n/projects/2048617/stories/167644917

Reply:
Is this story (and the related guidelines) still valid? I used intentionally the `TouchableOpacity` in order to provide a feedback to the user when press something.

Reply:
Btw removed in 3f397989833e271257cf2ab2ec745a4cd308a84a, we can discuss later this topic in order to have a common in app general behaviour.